patches/node/build_modify_js2c_py_to_allow_injection_of_original-fs_and_custom_embedder_js.patch

```diff
@@ -13,8 +13,29 @@ process and provide embedder modules (electrons
 renderer/browser/worker/sandboxed bootstrap scripts).  These are loaded
 through LoadEmbedderJavaScriptSource()
 
+diff --git a/lib/internal/fs/watchers.js b/lib/internal/fs/watchers.js
+index b45af42d12ff7df8a9e125e87f51af3456811c23..c84ff7feb07aebf656ada7e37d812d9d8a81300f 100644
+--- a/lib/internal/fs/watchers.js
++++ b/lib/internal/fs/watchers.js
+@@ -290,10 +290,12 @@ function emitCloseNT(self) {
+ 
+ // Legacy alias on the C++ wrapper object. This is not public API, so we may
+ // want to runtime-deprecate it at some point. There's no hurry, though.
+-ObjectDefineProperty(FSEvent.prototype, 'owner', {
+-  get() { return this[owner_symbol]; },
+-  set(v) { return this[owner_symbol] = v; }
+-});
++if (!'owner' in FSEvent.prototype) {
++  ObjectDefineProperty(FSEvent.prototype, 'owner', {
++    get() { return this[owner_symbol]; },
++    set(v) { return this[owner_symbol] = v; }
++  });
++}
+ 
+ async function* watch(filename, options = {}) {
+   const path = toNamespacedPath(getValidatedPath(filename));
```

Comment:
Is this the right patch file for this change to be in? the patch is titled "modify js2c" and doesn't touch other js files.

Reply:
@nornagon i'd say so - the issue is that the injections we make for js2c in this patch interact with circular dependencies in Node.js' `fs` implementation. So since `FSEvent` is shared we'd cause a failure since we'd try to make this property twice

Reply:
would be great to add this info to the patch description for easier research down the line when this breaks 🙃